patches/bootstrap.py

```diff
@@ -1,69 +1,104 @@
+from pathlib import Path
 import re
-import shutil
 
-import dash
 import pyodide
 import pyodide_js
+import micropip
 
-from pathlib import Path
 
-from xiplot.setup import setup_xiplot_dash_app
+JSBEAUTIFIER_VERSION = "THIS_VALUE_SHOULD_HAVE_BEEN_FILLED_WHEN_BUNDLING_XIPLOT"
+DASH_VERSION = "THIS_VALUE_SHOULD_HAVE_BEEN_FILLED_WHEN_BUNDLING_XIPLOT"
```

Comment:
Why was the explicit dash install added? I'm testing the PR right now and am sometimes experiencing race conditions in the installation - did you have something like this as well?

Reply:
Some of the dash javascript dependencies add the dash version to the file name. So this is to make sure that pyodide installs the version that corresponds to the files that were bundled. But as long as `pyproject.toml` only allows one specific version of dash, then this "fail-safe" is not strictly necessary.

Reply:
I haven't seen any race condition

Reply:
I see. I think the reason for specifying flask was that there were some issues with the testing xiplot on GitHub actions. But since that uses the `requirements.txt` anyway it might be best to remove it from the `pyproject.toml`.